src/duke/core/Duke.java

```diff
@@ -57,6 +65,37 @@ public void run() {
 
 
             switch(keyword[0]) {
+                case FIND:
+                    ArrayList<Task> result = searchTask(task, keyword[1]);
```

Comment:
Speaking of which, it would be better for you to reduce the code size of your methods and possibly implement a different class, so that your code for the FIND command will be more readable.